lib/redis/index.js

```diff
@@ -54,11 +55,36 @@ RedisClient.prototype.getLink = function(link) {
   })
 }
 
+RedisClient.prototype.fetchShortLinksFromTargetLinks = function(targetKeys) {
```

Comment:
We'll also need to expose and test `searchTargetLinks`. It can be a very lightweight test, not as exhaustive as that for `RedisClient.searchTargetLinks`.